application/helpers/common_helper.php

```diff
@@ -5087,3 +5087,23 @@ function recursive_preg_replace($pattern, $replacement, $subject, $limit = -1, &
     }
     return $result;
 }
+
+/**
+ * function to calculate the standard deviation of an array
+ * @var array $array The array to calculate the standard deviation for
```

Comment:
@var should be @param 

If it's an array of floats, can you document it as type `float[]`, please?